Source/Scene/Expression.js

```diff
@@ -52,6 +52,21 @@ define([
         setEvaluateFunction(this);
     }
 
+    function replaceVariables(expression) {
```

Comment:
Is this just replacing all instances of `${varName}` with the value of the varName property?  You can't do it with a single regex, but yes, it can be much much simpler.

Do we account for people that actually want `${varName}` in a string?  Can you escape the `$` with `\`?